RealifeTech/SDKConfiguration.swift

```diff
@@ -14,24 +14,30 @@ public struct SDKConfiguration {
     public let clientSecret: String
     public let apiUrl: String
     public let graphQLApiUrlString: String
+    public let webOrderingJourneyUrl: String
 
-    static let defaultApiUrl: String =  "https://api.livestyled.com/v3"
-    static let defaultGraphQLApiUrl: String = "https://graphql-eu.realifetech.com"
+    static let defaultApiUrl =  "https://api.livestyled.com/v3"
+    static let defaultGraphQLApiUrl = "https://graphql-eu.realifetech.com"
+    static let defaultWebOrderingJourneyUrl = "https://ordering.realifetech.com/"
 
-    /// - Parameter appCode: String used to identify the application which uses the RealifeTech SDK
-    /// - Parameter clientSecret: String which is used to authenticate with the RealifeTech backend
-    /// - Parameter apiUrl: [Optional] Provide a string value of a URL to override the SDK default
-    /// - Parameter graphApiUrl: [Optional] Provide a string value of a URL to override the SDK default
+    /// - Parameters:
+    ///   - appCode: String used to identify the application which uses the RealifeTech SDK
+    ///   - clientSecret: String which is used to authenticate with the RealifeTech backend
+    ///   - apiUrl: [Optional] Provide a string value of a URL to override the SDK default
+    ///   - graphQLApiUrlString: [Optional] Provide a string value of a URL to override the SDK default
+    ///   - webOrderingJourneyUrl: [Optional] Provide a string value of a URL to override the SDK default
     public init(
         appCode: String,
         clientSecret: String,
         apiUrl: String? = nil,
-        graphQLApiUrlString: String? = nil
+        graphQLApiUrlString: String? = nil,
+        webOrderingJourneyUrl: String? = nil
```

Comment:
Could you add String at the end please (`webOrderingJourneyUrlString`), in order to align the naming with `graphQLApiUrlString`

Reply:
No, it doesn't match the architecture. But I will rename `graphQLApiUrlString` as it's clear enough to know its type is String from documented comment.